src/components/sections/items/RadioSectionItem.tsx

```diff
@@ -22,7 +23,13 @@ type Props = SectionItemProps<{
   mode?: ActionModes;
   accessibility?: AccessibilityProps;
   color?: InteractiveColor;
+  rightAction?: {
+    icon: (props: SvgProps) => JSX.Element;
+    onPress: () => void;
+    isLoading?: boolean;
+  };
```

Comment:
Can this type be extracted into a variable and be reused both here and in RadioGroupSection?

Reply:
Btw is it a bit strange that in one place rightAction is a function, and in another an object? Perhaps the names should be slightly different?

Reply:
Good suggestion on the `itemToRightAction`, I'll do that!